index.js

```diff
@@ -1,81 +1,120 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+ console.log("%cQuestion 1", "color: red") 
+const counter = 1
+console.log(counter) 
+// => 2
+ console.log("%c----------", "color: red") 
```

Comment:
we want to be able to re-assign the counter variable which means that we should declare it with `let` instead of `const`. 

let counter = 1
counter = 2
console.log(counter)